src/components/menu/menu.stories.tsx

```diff
@@ -0,0 +1,228 @@
+                        <MenuItem onSelect={action('Edit')}>
+                            <StructuredMenuItem icon="✏" label="Edit" />
+                        </MenuItem>
+                        <MenuItem onSelect={action('Duplicate')}>
+                            <StructuredMenuItem icon="👯‍♀️️" label="Duplicate" shortcut="Cmd + D" />
+                        </MenuItem>
+                        <MenuItem
+                            onSelect={() => {
+                                action(flag ? 'Toggle off' : 'Toggle on')(flag ? '👎' : '👍')
+                                setFlag((f) => !f)
+                                return false // This prevents the menu from closing
+                            }}
+                        >
+                            <StructuredMenuItem
+                                icon={flag ? '👍' : '👎'}
+                                label="Toggle thumbs up or down"
+                            />
+                        </MenuItem>
+                        <hr />
+                        <MenuGroup label="Dangerous options">
+                            <MenuItem onSelect={action('Remove first')}>Remove first</MenuItem>
+                            <MenuItem onSelect={action('Remove completed')}>
+                                Remove completed
+                            </MenuItem>
+                            <MenuItem disabled onSelect={action('Remove all')}>
+                                Remove all (disabled)
+                            </MenuItem>
+                        </MenuGroup>
+                    </MenuList>
+                </Menu>
+            </Inline>
+        </section>
+    )
+}
+
+//
+// List of items, each with its own options menu
+//
+
+function OverflowIcon() {
+    return (
+        <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="16"
+            height="16"
+            fill="currentColor"
+            viewBox="0 0 16 16"
+        >
+            <path d="M3 9.5a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3z" />
+        </svg>
+    )
+}
+
+function Item({ value, name }: { value: string; name: string }) {
+    function getLabel(actionName: string) {
+        return `${actionName} '${name}'`
+    }
+    const ariaLabel = getLabel('Options menu for')
+    return (
+        <Columns as="li" border="standard" borderRadius="standard" padding="small" alignY="center">
+            <Column width="auto">{name}</Column>
+            <Column width="content">
+                <Menu onItemSelect={(itemValue) => action(`${String(itemValue)} '${name}'`)(value)}>
+                    <MenuButton
+                        as={Button}
+                        variant="quaternary"
+                        icon={<OverflowIcon />}
+                        aria-label={ariaLabel}
+                    />
+                    <MenuList aria-label={ariaLabel}>
```

Comment:
Should the `aria-label` be the same for both the `MenuButton` and `MenuList`?

Reply:
This is just for the storybook example. Is up to the consumer of these components to provide meaningful labels for each part.